### ISSUE TITLE
R0801: False positive on import exception

### ISSUE THREAD

**Issue by nickper** [PyCQA/pylint#8000]
### Bug description

The below config causes a R0801

file_A
try:
    import pexpect
    from pexpect import pxssh
    from tor_switch_object import TorSwitch
except ImportError as missing:
    raise ImportError(
        f"could not import all required Python modules: {missing}"
    ) from missing

file_B
try:
    from netmiko import ConnectHandler
    from tor_switch_object import TorSwitch
except ImportError as missing:
    raise ImportError(
        f"could not import all required Python modules: {missing}"
    ) from missing

### Command used

```shell
pylint src/*.py
```

### Pylint output

```shell
************* Module tor_switch_object
src/tor_switch_object.py:1:0: R0801: Similar lines in 2 files
==file_A:[10:17]
==file_B:[8:15]
    from tor_switch_object import TorSwitch
except ImportError as missing:
    raise ImportError(
        f"could not import all required Python modules: {missing}"
    ) from missing

 (duplicate-code)

------------------------------------------------------------------
Your code has been rated at 9.97/10 (previous run: 9.97/10, +0.00)
```

### Expected behavior

Your code has been rated at 10/10 (previous run: 9.97/10, +0.00)

### Pylint version

```shell
pylint 2.15.9
astroid 2.12.13
Python 3.10.8 (main, Dec 21 2022, 12:09:23) [GCC 9.4.0]
```

### OS / Environment

Ubuntu 20.04 in WSL2

**Comment by Pierre-Sassoulas** [comment]
Those 4 lines are really duplicated, so this is not a false positive. You could create a function, or remove the try except entirely instead (imho).

**Comment by nickper** [comment]
True on that. Let me clarify it more.
I don't find it logical that try-excepts should be taken into account for the duplicate-code. Because these are often the same or similar setup. for example the try-except block around the imports.

**Comment by Pierre-Sassoulas** [comment]
You can also change the ``min_similarity_lines`` value to 6 in order for the message to trigger less often, there's also a ``ignore_imports`` option.

**Comment by nickper** [comment]
Ok, will look at that. thanks
